src/vivarium_public_health/risks/distributions.py

```diff
@@ -96,12 +97,8 @@ def columns_created(self) -> list[str]:
         return [self._propensity]
 
     @property
-    def initialization_requirements(self) -> dict[str, list[str]]:
-        return {
-            "requires_columns": [],
-            "requires_values": [],
-            "requires_streams": [self._propensity],
-        }
+    def initialization_requirements(self) -> list[str | Resource]:
+        return [self.randomness]
```

Comment:
Should this be `self._propensity`?

Reply:
No that is the name of the stream and it needs to be the actual randomness stream